test/export/test_export.py

```diff
@@ -2331,6 +2331,20 @@ def forward(self, x):
         self.assertEqual(len(ep.graph_signature.input_specs), 4)
         self.assertTrue(torch.allclose(ep.module()(*inp), transform.module()(*inp)))
 
+    @testing.expectedFailureRetraceability
```

Comment:
Do you have the error message?

Reply:
No, but I can tell you it fails the assert that looks for lifted tensor constants (it has none). I think at some point I printed the exported program and it had an input buffer instead.